### ISSUE TITLE
Ch.4, p.39 - Weird example

### ISSUE THREAD

**Issue by olerass** [everydayrails/rspec_rails_4#27]
On page 39, line 5 we have the following:

> So far, if we wanted to specify a non-home phone in a spec we've done it like this:[code example follows]

There are two issues here:
- The example following this sentence is an updated version of the duplicate contact numbers test which uses FactoryGirl. However, the sentence refers to the fact that we have been using FactoryGirl all along which is confusing. Perhaps the problem lies in the fact that present tense is used; "we've done it like this"., which we of course haven't since this is a newly introduced example. A possible solution may be to use past tense, e.g. "we would have done it like this"
- The sentence talks about specifying non-home phone numbers and provides the following code as example, but the example code  doesn't specify non-home phone numbers.

To sum up: rephrase the sentence to focus on the fact that IF we wanted to use different phone number types, THEN it would look like the example. Also, use an example that actually illustrates the point by creating two different phone number types.

_Book version: 2013-10-29 (PDF)_

**Comment by JunichiIto** [comment]
> The sentence talks about specifying non-home phone numbers and provides the following code as example, but the example code doesn't specify non-home phone numbers.

I think so too.
